docs/api/portable-stories.md

```diff
@@ -0,0 +1,430 @@
+---
+title: 'Portable stories'
```

Comment:
Somehow we need to show that portable stories are only supported in React and Vue3 (for now). Not sure how to do that in the docs 🤔